techniques/system/common/1.0/promises.st

```diff
@@ -790,17 +790,16 @@ bundle edit_line enforce_content(str)
 bundle edit_line fix_syslogd(syslogd)
 {
   delete_lines:
-    any::
-      "^(local6)\s+(?!${syslogd}).*"
-        comment => "Delete missconfigured rudder syslogd destination";
-      "^local6\.\*\s+${syslogd}"
-        comment => "Delete old rudder syslogd format";
+
+      "^\s*local6.*"
```

Comment:
8 is not plenty, and some product (like Rudder) don't let the user choose the facility
Moreover the user won't notice until there is a problem. So I'm asking if there is not another solution. And if not we should make it more visible to the user.